scripts/python/services/test_GithubService.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from unittest.mock import call, MagicMock, patch
+
+from scripts.python.services.GithubService import GithubService
+
+ORGANISATION_NAME = "moj-analytical-services"
+
+
+class TestGithubService(unittest.TestCase):
+    mock_github = None
+
+    github_service = None
+
+    def setUp(self):
+        self.mock_github = MagicMock()
+        self.patcher = patch(target="github.Github.__new__",
```

Comment:
Managed to get this working after a bit of fiddling - not 100% convinced this is the cleanest approach. The decorator version seemed to look nicer, though I thought I'd not spend too much time playing with it 👀

Reply:
It is test code and it still make sense ie create new object. Do any of the lints or sonar cloud complain?

Reply:
No complaints, just felt a bit blegh - reading tests are hard enough - without random weird lines like `patch()' and `start.patch()/stop.patch()` 😂 

I only wrote it yesterday and can barely remember what they all do!

Felt like there'd be a cleaner solution. It might make itself clearer as I get more use to testing in python and the different frameworks 🖼️

Reply:
yeah I am not familiar with that terminology either, if the tests are working and linters are not complaining must be good